monorepo/website/src/pages/about/terms-of-use/terms-of-service.mdx

```diff
@@ -49,6 +49,8 @@ Submitters should ensure that they make their best efforts to provide good quali
 
 Submitters must ensure that any data submitted **does not contain human reads, clinically sensitive data**, or information that could **allow an individual to be publicly identified**. Users must not attempt to identify specific individuals from data within Pathoplexus. If you come across any data that may violate any of the above within Pathoplexus, please report it immediately to [submissions@pathoplexus.org](mailto:submissions@pathoplexus.org).
 
+Pathoplexus' aim is to provide pathogen sequences from epidemiological and biological settings (sequences which have a sample setting (human, animal, cell, culture) and sample date). In this regard, we do *not* accept sequences intended for vaccine production or patent purposes. Sequences that are clearly vaccine or patent sequences will be [revoked](/docs/how-to/revoke-sequences). We *do* accept vaccine-derived sequences (e.g. sequences obtained after vaccination).

```

Comment:
```suggestion
Pathoplexus' aim is to provide pathogen sequences from epidemiological and biological settings (sequences which have a sample setting (human, animal, cell, culture) and sample date). In this regard, we do *not* accept sequences that exist simply as in-silico or laboratory constructs, e.g. for patent purposes or for expression for vaccine production. Such synthetic sequences will be [revoked](/docs/how-to/revoke-sequences). We *do* accept vaccine-derived sequences (e.g. sequences obtained after vaccination).

```

Reply:
Yes, happy with this clarification - thanks Theo!
The only part I'm unsure of is "or for expression for vaccine production" - do you here mean like protein expression, or expression of intent? It reads a little awkwardly and the type of expression is unclear - maybe we could clarify?

Reply:
I meant protein expression, but yeah happy with any alternatives "laboratory use for vaccine production"

Reply:
(that's a bit closer to the ambiguity I was trying to avoid but probably ok)

Reply:
OK, I just deleted that second example - look OK now?

Reply:
(Outdated, I changed the TOS according to EB resolution.)